pkg/chains/storage/docdb/docdb.go

```diff
@@ -48,26 +47,21 @@ type SignedDocument struct {
 }
 
 // NewStorageBackend returns a new Tekton StorageBackend that stores signatures on a TaskRun
-func NewStorageBackend(ctx context.Context, logger *zap.SugaredLogger, tr *v1beta1.TaskRun, cfg config.Config) (*Backend, error) {
+func NewStorageBackend(ctx context.Context, logger *zap.SugaredLogger, cfg config.Config) (*Backend, error) {
 	url := cfg.Storage.DocDB.URL
 	coll, err := docstore.OpenCollection(ctx, url)
 	if err != nil {
 		return nil, err
 	}
 
-	return newStorageBackendWithColl(logger, tr, coll), nil
-}
-
-func newStorageBackendWithColl(logger *zap.SugaredLogger, tr *v1beta1.TaskRun, coll *docstore.Collection) *Backend {
 	return &Backend{
 		logger: logger,
-		tr:     tr,
 		coll:   coll,
-	}
+	}, nil
 }
 
 // StorePayload implements the Payloader interface.
-func (b *Backend) StorePayload(ctx context.Context, signed []byte, signature string, opts config.StorageOpts) error {
+func (b *Backend) StorePayload(ctx context.Context, tr *v1beta1.TaskRun, signed []byte, signature string, opts config.StorageOpts) error {
```

Comment:
nit: here and elsewhere, if the variable isn't being used use `_` as the variable name to make it clear that this isn't important for this func. 

```suggestion
func (b *Backend) StorePayload(ctx context.Context, _ *v1beta1.TaskRun, signed []byte, signature string, opts config.StorageOpts) error {
```

Reply:
Yes, that makes things clear! Thanks! 👍